src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -225,6 +225,19 @@ class ImageReplayerAdminSocketHook : public AdminSocketHook {
   Commands commands;
 };
 
+int replay_delay(const utime_t &event_time, int mirroring_delay) {
+    if (mirroring_delay <= 0) {
+      return 0;
+    }
+
+    utime_t now = ceph_clock_now(g_ceph_context);
```

Comment:
Minor: I think we want a new time helper to retrieving the UTC time, otherwise it would rely on users correctly setting the timezone offset and keeping it updated w/ daylight savings, etc.